src/directive/translate-cloak.js

```diff
@@ -20,7 +20,9 @@ angular.module('pascalprecht.translate')
  *                                  or hiding the cloak. Basically it relies on the translation
  *                                  resolve.
  */
-.directive('translateCloak', ['$rootScope', '$translate', function ($rootScope, $translate) {
+.directive('translateCloak', translateCloakDirective);
+
+function translateCloakDirective($rootScope, $translate) {
```

Comment:
Currently this is not acceptable for global functions. Before we can do this, we have to wrap the code with an IIFE.